schema/properties_directory/non-scalar/file.json

```diff
@@ -0,0 +1,44 @@
+{
+    "schemaId": "properties-directory-file",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "file property schema",
+    "properties": {
+        "name": {
+            "description": "name of the file",
+            "type": "string"
+        },
+        "type": {
+            "description": "type of the file",
+            "type": "string",
+            "enum": ["image", "text"]
+        },
+        "objectData": {
+            "description": "information describing the file and where it was run",
+            "type": "object",
+            "properties": {
+                "CONTAINER": {
+                    "description": "Container that ran the job resulting in the file",
```

Comment:
See if this is already defined elsewhere in ESSE

Reply:
Could not find this defined anywhere else. The closest thing is located in https://github.com/Exabyte-io/esse/blob/dev/schema/methods_directory/pseudopotential/file.json, under the "storage" property. It's still not quite the same; the "storage" property has keys for "URL", "Container", and "Provider."

Reply:
This is now defined in `core/reusable/object_storage_container_data.json` and is now referenced in the most recent commit.